### ISSUE TITLE
TF32 - we should enable this - or is it already done?

### ISSUE THREAD

**Issue by johnml1135** [sillsdev/machine.py#70]
https://huggingface.co/docs/transformers/perf_train_gpu_one#tf32

https://developer.nvidia.com/blog/accelerating-ai-training-with-tf32-tensor-cores/

```
import torch
torch.backends.cuda.matmul.allow_tf32 = True
torch.backends.cudnn.allow_tf32 = True
```

or

```
TrainingArguments(tf32=True, **default_args)
```

This appears to be a no-loss way to speed up training by a bit.  Is it already enabled?  I couldn't tell easily - though since this is 2 years old, it may be.

**Comment by johnml1135** [comment]
"TF32 mode is the default option for AI training with 32-bit variables on Ampere GPU architecture"
